technical-planning-meeting.md

```diff
@@ -3,39 +3,44 @@
+* When the technical implementation is not clear for a feature iteration
+* When undertaking work that requires multiple teams to collaborate on a technical solution
 
 ## Expected Outcomes
 
-* High level documentation of the business problem and technical solution
-* Developers should have a clear understanding of what is expected
-* A set of Jiras for product owners to prioritise
-* A definition of done
+* High level documentation of the business problem, the technical solution and/ or next steps and the business value (it is clear the complexity of the task in order that we can establish the business benefits)
+* Engineers should have a clear understanding of what is expected
+* What the next steps are, this could be a set of Jiras for Pod Leads to prioritise 
+* A definition of done/ acceptance criteria
+* Pod Leads or Stakeholders should be clear what Tech Debt there is with any MVP (Minimum Viable Product)
 
-## Prerequisites
 
-* 1 or more developers with a technical challenge
-* 1 or more software architects
-* 1 or more database gurus (if database work will be required)
-* 1 or more UXUI gurus (if frontend work will be required)
-* 0 or more business owners (if the business requirements are sufficiently complex)
-* 0 or more testers (if there is a large testing requirement)
-* 0 or more project gurus (if developers might require additional technical assistance)
+## Probable Attendees
+
+* Engineer(s) with a technical challenge
+* Engineer(s) to support the discussion and agree a technical solution/ next steps
+* Database gurus (if database work will be required)
+* UXUI gurus (if frontend work will be required)
+* Business owners (if the business requirements are sufficiently complex and to assess the business value)
+* Testers (if there is a large testing requirement)
+* Project gurus (engineers might require additional technical assistance)
 
 ## The Process
 
-1. The developers explain the business problem they're trying to solve and open it up for discussion.
-2. The developers explain their technical solution and open it up for discussion.
-3. We exchange thoughts and ideas, fill in any blanks and compromise where necessary to reach a good solution.
-4. We break the work down into small chunks.
-5. If the chunks of work are small enough to begin work, raise some Jiras. Otherwise, arrange for future technical planning meetings to dig deeper into the problem (an example of when this might be necessary - "We want to start selling Coach")
-6. The developers should leave the meeting and produce a markdown document covering everything discussed in parts #1 to #4
-7. Product owners prioritise the work as appropriate
+1. Make a copy of the [TPM template](https://docs.google.com/a/holidayextras.com/document/d/1zVbOz0dRAnzZ6UhjYO1Ce1YGI2dzvEoYvZfkTK7qjYU/edit?usp=sharing). Part 1 is completed and to increase visibility that the TPM is taking place, notification of the TPM is posted with a link to the document in the Slack Channel #tpm
```

Comment:
This might be a good opportunity to group this into:
- Before
- During
- After